core/object.cpp

```diff
@@ -435,93 +435,212 @@ void Object::set(const StringName &p_name, const Variant &p_value, bool *r_valid
 
 	} else if (p_name == CoreStringNames::get_singleton()->_meta) {
 		//set_meta(p_name,p_value);
-		metadata = p_value;
-		if (r_valid)
-			*r_valid = true;
-		return;
+metadata = p_value;
```

Comment:
You broke indentation all over the file :)
See http://docs.godotengine.org/en/latest/community/contributing/code_style_guidelines.html#using-clang-format-locally

Reply:
oh gosh. Yeah, thanks for this. Hopefully I can fix it up. XD